modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/AppDatabase.kt

```diff
@@ -989,6 +997,45 @@ abstract class AppDatabase : RoomDatabase() {
             database.execSQL("ALTER TABLE podcasts ADD COLUMN funding_url TEXT")
         }
 
+        val MIGRATION_113_114 = addMigration(113, 114) { database ->
+            with(database) {
+                beginTransaction()
+                try {
+                    database.execSQL(
+                        """
+                        CREATE TABLE IF NOT EXISTS notification(
+                            _id INTEGER PRIMARY KEY AUTOINCREMENT,
+                            category INTEGER NOT NULL,
+                            subcategory TEXT NOT NULL,
+                            deeplink TEXT NOT NULL,
+                            trigger_rule INTEGER NOT NULL,
+                            trigger_value TEXT
+                        )
+                        """.trimIndent(),
```

Comment:
I might not fully understand this but do we need to store all the information about the notifications in the database? Should we only store the information the user can change in the database such as if they have toggled on the notification and the last sent time.

Reply:
My idea was to have one place to organize the notifications. We will have different type of notifications with different rules and frequency. So I was thinking to have them in one place

Reply:
I am going to remove the notification's details to move to some sealed classes

Reply:
Maybe we can leave this discussion until more of the feature is complete but I wonder if we even need the `notifications` table in the database as it will have the same values for every user. Unless I misunderstand, that will be stored in the table.